config/config.apps.sample.php

```diff
@@ -339,19 +343,26 @@
 		'picture-claim' => 'picture',
 		  // defines a list of groups to which the newly created user will be added automatically
 		'groups' => ['admin', 'guests', 'employees']
-	  ],
+	],
+	  // `mode` and `search-attribute` will be used to create a unique use in ownCloud
```

Comment:
```suggestion
	// `mode` and `search-attribute` will be used to create a unique user in ownCloud
```

IMO this is meant to be "user"